elasticsearch/rest/src/test/java/org/apache/metamodel/elasticsearch/rest/ElasticSearchRestDataContexFactoryIT.java

```diff
@@ -55,22 +63,24 @@
     public void setUp() throws Exception {
         dockerHostAddress = ElasticSearchRestDataContextIT.determineHostName();
 
-        externalClient = new ElasticSearchRestClient(RestClient.builder(new HttpHost(dockerHostAddress, 9200)).build());
+        externalClient = new RestHighLevelClient(RestClient.builder(new HttpHost(dockerHostAddress, 9200)));
```

Comment:
9200 is actually used on multiple locations in this module. A constant would be nice.